apps/root/src/pages/strategy-guardian-detail/strategy-management/index.tsx

```diff
@@ -53,7 +55,11 @@ const StrategyManagement = ({ chainId, strategyGuardianId }: StrategyManagementP
           }
         />
       </UnderlinedTabs>
-      {tab === 0 && <DepositForm strategy={strategy} setHeight={setHeight} />}
+      <ContainerBox gap={tab === 0 ? 3 : 5} flexDirection="column">
```

Comment:
This should be the same, if it is not we need to flag the designer

Reply:
Checking!

Reply:
Having two inputs in the Container, requires a different gap (from design perspective). They had some outdated styles also for the Paper that holds both withdraw and deposit forms, so I included it in the last commit